docs/how-to/ccv0.sh

```diff
@@ -199,26 +204,32 @@ build_and_install_kata_runtime() {
 }
 
 configure() {
+    local containerd_config="/etc/containerd/config.toml"
+    export RUNTIME_CONFIG_PATH=/etc/kata-containers/configuration.toml
+
     debug_function configure_kata_to_use_rootfs
     debug_function enable_full_debug
     # Temp PoC verify code: Inject policy path config parameter
-    sudo sed -i -e 's%^kernel_params = "\(.*\)"%kernel_params = "\1 agent.container_policy_file=/etc/containers/quay_verification/quay_policy.json"%g' /etc/kata-containers/configuration.toml
+    sudo sed -i -e 's%^kernel_params = "\(.*\)"%kernel_params = "\1 agent.container_policy_file=/etc/containers/quay_verification/quay_policy.json"%g' ${RUNTIME_CONFIG_PATH}
 
     # If using AA then need to add the agent_config
     if [ "${AA_KBC}" == "offline_fs_kbc" ]; then
-        sudo sed -i -e 's%^kernel_params = "\(.*\)"%kernel_params = "\1 agent.config_file=/etc/agent-config.toml"%g' /etc/kata-containers/configuration.toml
+        sudo sed -i -e 's%^kernel_params = "\(.*\)"%kernel_params = "\1 agent.config_file=/etc/agent-config.toml"%g' ${RUNTIME_CONFIG_PATH}
     fi
 
-    # insert the cri_handler = "cc" into the [plugins.cri.containerd.runtimes.kata] section
-    sudo sed -z -i 's/\([[:blank:]]*\)\(runtime_type = "io.containerd.kata.v2"\)/\1\2\n\1cri_handler = "cc"/' /etc/containerd/config.toml
+    # insert the cri_handler = "cc" into the [plugins.cri.containerd.runtimes.kata] section if missing
+    sudo grep -qF "cri_handler = \"cc\"" ${containerd_config} || \
+    sudo sed -z -i 's/\([[:blank:]]*\)\(runtime_type = "io.containerd.kata.v2"\)/\1\2\n\1cri_handler = "cc"/' ${containerd_config}
 
     # Add cni directory to containerd config
+    sudo grep -qF "conf_dir = \"/etc/cni/net.d\"" ${containerd_config} || \
     echo "    [plugins.cri.cni]
       # conf_dir is the directory in which the admin places a CNI conf.
-      conf_dir = \"/etc/cni/net.d\"" | sudo tee -a /etc/containerd/config.toml
+      conf_dir = \"/etc/cni/net.d\"" | sudo tee -a ${containerd_config}
     
     # Switch image offload to true in kata config
-    sudo sed -i -e 's/^# *\(service_offload\).*=.*$/\1 = true/g' /etc/kata-containers/configuration.toml
+    source ${BATS_TEST_DIRNAME}/lib.sh
```

Comment:
Can you replace the 4 function-local `source` calls with a single top-level `source` maybe around line 51 where we source `/etc/os-release`?